package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package>
+  <name>h264_image_transport</name>
+  <version>0.0.0</version>
```

Comment:
Bump to 0.0.1.

Reply:
done

Reply:
This doesn't seem to be done - forgot to commit?

Reply:
done